jbehave-support-core/src/main/java/org/jbehavesupport/core/splunk/SplunkConfig.java

```diff
@@ -0,0 +1,47 @@
+package org.jbehavesupport.core.splunk;
+
+import com.splunk.SSLSecurityProtocol;
+import com.splunk.ServiceArgs;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NonNull;
+import org.springframework.stereotype.Component;
+
+import java.util.Objects;
+
+@Data
+@Builder
+@AllArgsConstructor
+@Component
+public class SplunkConfig {
```

Comment:
It just occured to me - shouldnt we allow multiple splunk configs for multiple servers (distinguished by qualifier or something)? AFAIK in EIT there are several local splunk servers which are then aggregated in the central one, do we want to allow just one splunk connection or multiple? (though I guess real world usage will be just one splunk server)

Reply:
As discussed with Pavel, we will stick currently with a single Splunk instance. Should there be a need to have it env specific, guys can use Spring profile.